src/main/java/net/dv8tion/jda/api/hooks/ListenerAdapter.java

```diff
@@ -143,6 +141,9 @@ public void onUserContextInteraction(@Nonnull UserContextInteractionEvent event)
     public void onMessageContextInteraction(@Nonnull MessageContextInteractionEvent event) {}
     public void onButtonInteraction(@Nonnull ButtonInteractionEvent event) {}
     public void onSelectMenuInteraction(@Nonnull SelectMenuInteractionEvent event) {}
```

Comment:
We might want to consider renaming the original "SelectMenu" to "StringSelectMenu" as that's what it is now considered.
@MinnDevelopment opinion?

Reply:
Seems like a good idea to me